crates/re_types/definitions/rerun/components.fbs

```diff
@@ -22,7 +22,6 @@ include "./components/position2d.fbs";
 include "./components/position3d.fbs";
 include "./components/radius.fbs";
 include "./components/range1d.fbs";
-include "./components/range2d.fbs";
```

Comment:
Probably missing something but why isn't `components.VisualBounds2D` added to this list?

Reply:
it's on the blueprint.fbs